olioapps/aframe-typescript-toolkit
# Inline docs

### WHAT 
1. add examples to componentWrapper, systemWrapper, and entity builder docs
![image](https://user-images.githubusercontent.com/15587168/46555548-8793e180-c898-11e8-8b68-1c992cfc59c4.png)

### TEST 
1. run `yarn install && yarn build`
2. serve docs `cd dist/docs && python -m SimpleHTTPServer 3001`
3. view the docs at http://localhost:3001/
- component wrapper http://localhost:3001/classes/_aframe_wrapper_.componentwrapper.html
- system wrapper http://localhost:3001/classes/_aframe_wrapper_.systemwrapper.html
- entity builder http://localhost:3001/classes/_entity_builder_.entitybuilder.html
4. See that docs for entity builder, component wrapper, and scene wrapper are looking good
